.travis/install_deps.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+sudo apt-get update
+wget https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh -O miniconda.sh;
+bash miniconda.sh -b -p $HOME/miniconda
+export PATH="$HOME/miniconda/bin:$PATH"
+hash -r
+conda config --set always_yes yes --set changeps1 no
+conda update -q conda
+  # Useful for debugging any issues with conda
+conda info -a
+conda create -q -n test-environment pytorch cpuonly python=$TRAVIS_PYTHON_VERSION -c $PYTORCH_CHANNEL
+source activate test-environment
```

Comment:
`source` wont work externally, so when you call pytest etc conda env is not activated.
I propose to change that and install everything in `base` conda env. Thus, we do not have to create/activate another env.